boards/sensor_v2/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "sensor_v2"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+fugit = "0.3.5"
+embedded-hal = { version = "0.2.6", features = ["unproven"] }
+embedded-dma = "0.2.0"
+panic-halt = "0.2.0" 
+cortex-m = { workspace = true }
+cortex-m-rt = "^0.7.0"
+cortex-m-rtic = "1.1.3"
+#common-arm = {path = "../../libraries/common-arm"}
```

Comment:
Is there a conflict with the common-arm memory.x file when this is included? Doesn't appear to cause issues when building with all boards.